1.py

```diff
@@ -0,0 +1,121 @@
+import sqlite3
+import json
+
+from elasticsearch import Elasticsearch
+from elasticsearch.helpers import bulk
+
+
+def extract():
+    """
+    extract data from sql-db
+    :return:
+    """
+    connection = sqlite3.connect("db.sqlite")
+    cursor = connection.cursor()
+
+    # Наверняка это пилится в один sql - запрос, но мне как-то лениво)
+
+    # Получаем все поля для индекса, кроме списка актеров и сценаристов, для них только id
+    cursor.execute("""
+        select id, imdb_rating, genre, title, plot, director,
+        -- comma-separated actor_id's
+        (
+            select GROUP_CONCAT(actor_id) from
+            (
+                select actor_id
+                from movie_actors
+                where movie_id = movies.id
+            )
+        ),
+        
+        max(writer, writers)
+        from movies
+    """)
+
+    raw_data = cursor.fetchall()
+
+    # cursor.execute('pragma table_info(movies)')
+    # pprint(cursor.fetchall())
+
+    # Нужны для соответсвия идентификатора и человекочитаемого названия
+    actors = {row[0]: row[1] for row in cursor.execute('select * from actors where name != "N/A"')}
+    writers = {row[0]: row[1] for row in cursor.execute('select * from writers where name != "N/A"')}
+
+    return actors, writers, raw_data
+
+
+def transform(__actors, __writers, __raw_data):
+    """
+
+    :param __actors:
+    :param __writers:
+    :param __raw_data:
+    :return:
+    """
+    documents_list = []
+    for movie_info in __raw_data:
+        # Разыменование списка
+        movie_id, imdb_rating, genre, title, description, director, raw_actors, raw_writers = movie_info
+
+        if raw_writers[0] == '[':
```

Comment:
startswith позволит избежать ошибки `IndexError`
```suggestion
        if raw_writers.startswith('['):
```